feconf.py

```diff
@@ -470,6 +470,12 @@ def get_empty_ratings():
 # A sorted list of default categories.
 DEFAULT_CATEGORIES = sorted(CATEGORIES_TO_COLORS.keys())
 
+#Size of the gravatar in px assigned to the user after signup
+GRAVATAR_SIZE_PX = 150
+
+#Default Identicon for new users
+DEFAULT_IDENTICON_DATA_URL = '/images/avatar/user_blue_72px.png'
```

Comment:
Actually, thinking about it more, you can probably put both these constants at the top of user_services.py, since they're really only used there.

I also think that this should be the data URL of an actual identicon. You could do the following:
- create a 'default' identicon and put the PNG in /static/images/avatar
- create a constant in user_services.py with the data URL, and add a comment pointing to the PNG location
- add a test in user_services_test.py that checks that runs utils.convert_png_to_data_url() on the PNG and verifies its equality with the constant in user_services.py.